crates/astria-sequencer/src/sequencer.rs

```diff
@@ -98,3 +104,47 @@ impl Sequencer {
         Ok(())
     }
 }
+
+fn start_grpc_server(
```

Comment:
Should we have create, start, and stop functions on the server? Something akin to what we do in go execution API server? https://github.com/astriaorg/go-ethereum/pull/36/files#diff-cf421ae83f75df74a828a831be8b14967d79fb76fd0703606d804724e32d832f

Reply:
just wondering why? we don't have start/stop methods on the existing ABCI server

Reply:
Specifically with grpc because grpc supports streaming, even if these apis don't and ideally want to close those cleanly.

In general we want to move towards having a graceful shutdown though, and closing the server is a part of that.